src/main/java/Duke.java

```diff
@@ -1,10 +1,126 @@
+import java.util.Scanner;
+
 public class Duke {
+    static final String LINE = "____________________________________________________________";
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String inputString = "";
+        Task[] tasks = new Task[100];
+        int tasksCount = 0;
+        Scanner in = new Scanner (System.in);
+
+        welcomeMessage();
+
```

Comment:
This while loop can be changed to a separate method outside the main method, which will make the main method much shorter